lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -388,6 +393,44 @@ def test_invalid_logout(client, server_config):
             )
             assert response.status_code == HTTPStatus.OK
 
+    @staticmethod
+    def test_forgot_password_valid_email(client, server_config):
+        with client:
+            # user registration
+            resp_register = register_user(
+                client,
+                server_config,
+                username="username",
+                firstname="firstname",
+                lastname="lastName",
+                email="user@domain.com",
+                password="12345",
+            )
+            assert resp_register.status_code == HTTPStatus.CREATED
+
+        resp_forgot_password = forgot_password(client, server_config, "user@domain.com")
+        assert resp_forgot_password.status_code == HTTPStatus.OK
+
+    @staticmethod
+    def test_forgot_password_invalid_email(client, server_config):
+        with client:
+            # user registration
+            resp_register = register_user(
+                client,
+                server_config,
+                username="username",
+                firstname="firstname",
+                lastname="lastName",
+                email="user@domain.com",
+                password="12345",
+            )
+            assert resp_register.status_code == HTTPStatus.CREATED
+
+        resp_forgot_password = forgot_password(
+            client, server_config, "user1@domain.com"
```

Comment:
You should probably mock out the `flask_mail` `send()` method for this test too -- the mock should cause the test to fail if it _is_ called.